### DISCUSSION TITLE
How to update a user's `active` status

### DISCUSSION THREAD

**Discussion by daniel-anya** [apache/airflow#30101] [Q&A]
Hi all

I'm trying to deactivate a user via the [PATCH User endpoint](https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/patch_user) but I get a `400` status code with the error message - `"Property is read-only - 'active'"`. I see [here](https://github.com/apache/airflow/blob/0918223/airflow/api_connexion/schemas/user_schema.py#L41) that the `active` property is intentionally set to `read-only` so I'm curious why this is the case and if there's a recommended way to update a user's active status.

**Comment by daniel-anya** [comment]
@potiuk Wondering if you have any insight here? 🙏

**Reply by potiuk** [reply]
no idea
